.github/workflows/build.yml

```diff
@@ -0,0 +1,98 @@
+
+name: Build
+on:
+  push:
+    branches:
+    - main
+    - 'release-*'
+    tags:
+    - 'v*'
+    tags-ignore:
+    - dev
```

Comment:
Could this have a `pull_request:` added the `release-*` branches as well? Additionally it should be ok to remove `release-*` from the `branches:` to avoid the double builds for PRs and release branches on merge.